client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -234,6 +244,7 @@ export class RegistrantExtraInfoCaForm extends React.PureComponent {
 
 export default connect(
 	state => ( {
+		contactDetails: getContactDetailsCache( state ),
 		contactDetailsExtra: getContactDetailsExtraCache( state ),
```

Comment:
`contactDetailsExtra` is just `contactDetails.extra`, so we might want to drop it. I've been wondering about dropping it everywhere, actually, so let me know if you made a deliberate choice to keep it.

Reply:
Good call. I left it in there, well, because it was there. Will update. 🙇